timemachine/fe/rbfe.py

```diff
@@ -99,9 +99,9 @@ def sample(initial_state, protocol):
     intg_impl = initial_state.integrator.impl()
     baro_impl = initial_state.barostat.impl(bound_impls)
 
-    # minimize the local region, removing clashes and unstretching some bonds
-    cutoff = 1.0  # in nanometers
-    # local minimize region around ligand
+    cutoff = 0.5  # in nanometers
+    # local minimize region around ligand, should result in ~300 atoms
```

Comment:
Possible alternatives to consider in the future:
* If we want to directly control the number of atoms, we could sort by distance to ligand, and take the top `K=300` atoms, rather than taking all atoms within a radius
* Could modify the selection to include all clashy atoms, rather than atoms based on distance to a ligand assumed to be the sole source of clashes